third_party/terraform/website/docs/r/google_kms_crypto_key_iam.html.markdown

```diff
@@ -169,8 +169,13 @@ The `condition` block supports:
   
 ## Attributes Reference
 
-In addition to the arguments listed above, the following computed attributes are
-exported:
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `id` - an identifier for the resource google_kms_crypto_key_iam_member with format `projects/{{project}}/locations/{{location}}/keyRings/{{key-ring-name}}/cryptoKeys/{{key-name}}/roles/{{role}}/user:foo@example.com`
```

Comment:
I don't think that we want to document the `id` field on IAM resources. This is because IAM resources (bindings and members) are sub resources of IAM policies, and so the id cannot refer to a real GCP resource. The idea behind the id change was that we could replace uses of .self_link with .id, but the id for IAM resources can never be used like that